### PR TITLE
Fix extra can't redefined dependency version

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/source/installation.rst [hunk 1]
```diff
@@ -126,10 +126,13 @@ the ``-d`` or ``--dir=`` argument.
 6. Include in application
 .........................
 
-To include ``pyramid_oereb`` into your existing Pyramid application, you first have to include the plugin in
-your application's main method.
+In your ``setup.py`` you shouldn't directly depends on ``pyramid_oereb`` because you will not depends on any
```
Comment (jwkaltz) [inline doc/source/installation.rst]: I don't understand the second part of the sentence.

File: doc/source/installation.rst [hunk 2]
```diff
@@ -126,10 +126,13 @@ the ``-d`` or ``--dir=`` argument.
...
+other package. You should use ``pyramid_oereb[recommend]`` to gets the recommended versions of the
```
Comment (jwkaltz) [inline doc/source/installation.rst]: ... "to get the recommended versions of the dependencies, or ``pyramid_oereb[no-version]`` to get the dependencies without specifying which versions of these dependencies shall be used."

File: doc/source/installation.rst (hunk 1 above)
Comment (sbrunner) [reply doc/source/installation.rst]: ... what don't you understand?

Comment (jwkaltz) [reply doc/source/installation.rst]: the "because you will not depends on any..." part.

File: doc/source/installation.rst (hunk 2 above)
Comment (sbrunner) [reply doc/source/installation.rst]: Done :-)

**Comment by sbrunner** [comment]
Unfortunately no, if we do `pip install pyramid_oereb` we don't have any dependency, we should do 
`pip install pyramid_oereb[recommend]` or `pip install pyramid_oereb[no-version]` for me pip is not as flexible as I want ...

File: doc/source/installation.rst (hunk 1 above)
Comment (jwkaltz) [reply doc/source/installation.rst]: Can we inverse the sentence?

Comment (sbrunner) [reply doc/source/installation.rst]: Fixed :-)

**Pull request merged**